### ISSUE TITLE
suspected vulnerability of the metrics port 9101

### ISSUE THREAD

**Issue by grassets-tech** [aptos-labs/aptos-ait2#33]
Issue statement:
We have noticed significant Tx increase utilising full 1Gbs network bandwidth since the latest changes applied about 12 hours ago. RX packets 1882428703 bytes 5192380941113 (5.1 TB) --> IP 129.151.73.134 using wide range of ports.

The latest changes were to update to the image testnet_f5d8013b0a1851da8e078394d83130d3adaf7670 and setting the timeout to 2000ms.

On diagram below you can see Tx increase at around 8:30 UTC+1
![image](https://user-images.githubusercontent.com/82155440/180145719-880bc574-8a25-4e4e-b7d6-08f4132bb1e8.png)

Workaround solution:

close metric port 9101.
On diagram below you can see Tx is back to normal at around 23:30 UTC+1

![image](https://user-images.githubusercontent.com/82155440/180145806-eb8183da-d2ff-45d3-a5bb-6778bfeb5f57.png)

**Comment by sherry-x** [comment]
Yes, we did get several report that people getting attacked on metrics port, please make sure you keep the port closed. This is part of our documentation for node best practice
